aws-java-sdk-core/src/test/java/com/amazonaws/util/StringUtilsTest.java

```diff
@@ -22,22 +22,28 @@
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNull;
 import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertThat;
 
+import java.math.BigDecimal;
+import java.math.BigInteger;
 import java.nio.ByteBuffer;
-import java.util.Locale;
+import java.util.Date;
 import java.util.Random;
 
 import org.hamcrest.Matchers;
-import org.hamcrest.core.IsEqual;
-import org.hamcrest.text.IsEmptyString;
-import org.junit.Assert;
+import org.junit.Rule;
```

Comment:
Why are you removing their unused imports? Cover doesn't do this.

Reply:
Reverted changes